### ISSUE TITLE
Listener not triggered

### ISSUE THREAD

**Issue by camueller** [mattjlewis/pigpioj#16]
I'm using `PigpioInputTest` you provide. It configures BCM 12 for input and registers itself as listener for callbacks.
If I connect BCM 12 with +5V it should display "Callback ...." but it doesn't. Instead there is an error messgae `Error, notification handle not set`.

Complete output:
```
$ java -DPIGPIOD_HOST=localhost -cp pigpioj-java-2.6.0.jar:netty-transport-4.1.69.Final.jar:netty-common-4.1.69.Final.jar:netty-codec-4.1.69.Final.jar:netty-resolver-4.1.69.Final.jar:netty-buffer-4.1.69.Final.jar uk.pigpioj.test.PigpioInputTest
version: 79
Dec 23, 2021 9:56:47 AM uk.pigpioj.PigpioSocket enableListener
WARNING: Error, notification handle not set
Sleeping for 20s
Dec 23, 2021 9:57:08 AM uk.pigpioj.PigpioSocket disableListener
WARNING: Error, notification handle not set
```

I have `pigpiod` running.
The `PigpioOutputTest` runs just fine:
```
$ java -DPIGPIOD_HOST=localhost -cp pigpioj-java-2.6.0.jar:netty-transport-4.1.69.Final.jar:netty-common-4.1.69.Final.jar:netty-codec-4.1.69.Final.jar:netty-resolver-4.1.69.Final.jar:netty-buffer-4.1.69.Final.jar uk.pigpioj.test.PigpioOutputTest
version: 79
on
off
on
off
on
off
on
off
on
off
pwm full on
pwm full off
pwm 25% off
pwm 50%
pwm 75%
pwm full off
```

**Comment by mattjlewis** [comment]
Just looking into this - I can reproduce the error locally. The bug seems to have been introduced with the changes I made when moving to Netty 4.1.6x.

**Comment by mattjlewis** [comment]
Fixed in 2.6.1

**Comment by camueller** [comment]
Thank's a lot for this very quick fix. I'm mirgating to pigpio from Pi4J which I used for 5 years.
